src/styles/card.scss

```diff
@@ -0,0 +1,110 @@
+.card {
+  width: 200px;
+  padding: 15px 14px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-family: "Roboto", sans-serif;
+
+  &__photo {
+    width: 160px;
+    height: 134px;
+    margin: 15px 5px 42px;
+  }
+
+  &__name {
+    margin: 0;
+    margin-bottom: 5px;
+
+    line-height: 15px;
+    font-size: 12px;
+    font-weight: 500;
+  }
+
+  &__code {
+    line-height: 13px;
+    font-size: 11px;
+  }
+
+  &__rating {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-top: 15px;
+    margin-bottom: 20px;
+  }
+
+  &__stars--4 {
+    display: flex;
+  }
+
+  .stars__star {
+    background: url(../images/star.svg) no-repeat center;
+    width: 15px;
+    height: 15px;
+    margin-right: 4px;
+  }
+
+  .stars__star:last-child {
+    margin-right: 0;
+  }
+
+  &__stars--1 .stars__star:nth-child(-n+1),
```

Comment:
You can use an abbreviation in the element reference to avoid duplicate code. 

This will apply the given style to all elements that have a class starting with "stars__star", and is a more compact and readable way to customize styles for active stars. 

`[class^="stars__star"] {
  background-image: url(../images/star-active.svg);
}
`